submissions/NatalyaBortsova/popup/index.html

```diff
@@ -0,0 +1,177 @@
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link animation animation_icon">
+                      <img src="images/icons/news.png" alt="News" width="50" height="50" />
+                      <span class="popup__name">News</span>
+                    </a>
+                  </li>
+                </ul>
+                <div class="popup__block">
+                  <input class="menu__item-checkbox menu__item-checkbox_more" id="more" type="checkbox">
+                  <span class="menu__item-check menu__item-check_more" tabindex="0"></span>
+                  <label class="menu__item-label menu__item-label_more" for="more" tabindex="0">More</label>
+                  <ul class="popup__list popup__list_more">
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/photos.png" alt="Photos" width="50" height="50" />
+                        <span class="popup__name">Photos</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/promo.png" alt="Promo" width="50" height="50" />
+                        <span class="popup__name">Promo</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/chat.png" alt="Translation" width="50" height="50" />
+                        <span class="popup__name">Translation</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/videos.png" alt="Videos" width="50" height="50" />
+                        <span class="popup__name">Videos</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/blogs.png" alt="Blogs" width="50" height="50" />
+                        <span class="popup__name">Blogs</span>
+                      </a>
+                    </li>
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/search.png" alt="Search" width="50" height="50" />
+                        <span class="popup__name">Search</span>
+                      </a>
+                    </li>
+                  </ul>
+                  <a class="popup__bottom-link animation animation_icon" tabindex="0">
```

Comment:
By default all interactive elements have focus, `tabindex="0"` doing the same as tag `a` without tabindex attribute)